src/common/checksum.c

```diff
@@ -1045,6 +1058,17 @@ csum_chunk_align_ceiling(daos_off_t off, size_t chunksize)
 	return hi;
 }
 
+daos_off_t
+csum_record_chunksize(daos_off_t default_chunksize, daos_off_t rec_size)
+{
+	D_ASSERT(rec_size > 0 && default_chunksize > 0);
+	if (default_chunksize % rec_size == 0)
+		return default_chunksize;
```

Comment:
The first "if" branch is redundant, it can be covered by the subsequent "(default_chunksize / rec_size) * rec_size".

Reply:
Good catch. Fixed.